tests/nnstreamer_grpc/unittest_grpc.cc

```diff
@@ -0,0 +1,447 @@
+/**
+ * @file    unittest_grpc.cc
+ * @date    28 Oct 2020
+ * @brief   Unit test for gRPC tensor source/sink plugin
+ * @see     https://github.com/nnstreamer/nnstreamer
+ * @author  Dongju Chae <dongju.chae@samsung.com>
+ * @bug     No known bugs
+ */
+
+#include <gtest/gtest.h>
+#include <glib.h>
+
+#include <nnstreamer_plugin_api_filter.h>
+#include <nnstreamer_plugin_api.h>
+
+#define NNS_GRPC_PLUGIN_NAME "nnstreamer_grpc"
+#define NNS_GRPC_TENSOR_SRC_NAME "tensor_src_grpc"
+#define NNS_GRPC_TENSOR_SINK_NAME "tensor_sink_grpc"
+
+/**
+ * @brief Test gRPC tensor_src/sink existence.
+ */
+TEST (nnstreamer_grpc, check_existence)
+{
+  GstPlugin *plugin;
+  GstElementFactory *factory;
+
+  plugin = gst_registry_find_plugin (gst_registry_get (),
+      NNS_GRPC_PLUGIN_NAME);
+  EXPECT_TRUE (plugin != NULL);
+  gst_object_unref (GST_OBJECT (plugin));
```

Comment:
```suggestion
  gst_object_unref (plugin);
```
Casting to ```GST_OBJECT``` looks not necessary.
https://github.com/GStreamer/gstreamer/blob/master/gst/gstplugin.c

Reply:
Right, thanks.